src/UserLabel/UserLabel.stories.tsx

```diff
@@ -31,7 +31,7 @@ const Template: StoryFn<UserLabelProps> = args => {
 export const Default = {
   args: {
     label: "James Harper",
-    tooltip: ""
+    tooltip: "Updated at"
```

Comment:
This is the UserLabel

```suggestion
    tooltip: "Created by"
```